knative-operator/pkg/controller/knativeserving/consoleclidownload/consoleclidownload.go

```diff
@@ -12,24 +12,26 @@ import (
 	mf "github.com/manifestival/manifestival"
 	consolev1 "github.com/openshift/api/console/v1"
 	routev1 "github.com/openshift/api/route/v1"
-	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/equality"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
 	servingv1alpha1 "knative.dev/operator/pkg/apis/operator/v1alpha1"
+	servingv1 "knative.dev/serving/pkg/apis/serving/v1"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 const (
-	knCLIDownload                   = "kn"
-	knDownloadServer                = "kn-download-server"
-	knConsoleCLIDownloadDeployRoute = "kn-cli-downloads"
+	knCLIDownload               = "kn"
+	knDownloadServer            = "kn-download-server"
+	knConsoleCLIDownloadService = "kn-cli-downloads"
 )
 
 var log = common.Log.WithName("consoleclidownload")
+var knServiceGVK = schema.GroupVersionKind{Group: "serving.knative.dev", Version: "v1", Kind: "Service"}
```

Comment:
This is unused and more correctly constructed below now.

Reply:
It's a leftover from my unstructured PoC impl, I'll remove it, thanks.